app/controllers/backend/accounts_controller.rb

```diff
@@ -97,6 +97,11 @@ def self.account_moves_conditions(_options = {})
       t.column :supplier_account, url: true
     end
 
+    list(:product_nature_variants, joins: :category, conditions: ['? IN (product_nature_categories.charge_account_id, product_nature_categories.product_account_id, product_nature_categories.fixed_asset_account_id, product_nature_categories.stock_account_id)', 'params[:id]'.c], order: { created_at: :desc }) do |t|
```

Comment:
A new action must be configured in `config/navigation.xml` and `config/rights`. See `rake clean`.